virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc

```diff
@@ -0,0 +1,12 @@
+[id="virt-reclaiming-persistent-volumes"]
+= Reclaiming persistent volumes
+include::modules/virt-document-attributes.adoc[]
+:context: virt-reclaiming-persistent-volumes
+toc::[]
+
+When a statically-provisioned persistent volume (PV) is unbound from a
```

Comment:
Rewrite in active voice.

Reply:
Changed to:

Perform volume _reclaimation_ when a statically-provisioned persistent volume (PV) 
is unbound from a persistent volume claim (PVC), to bind the PV to a new PVC for reuse.